common/pom.xml

```diff
@@ -69,6 +69,11 @@
       <groupId>org.jboss.logging</groupId>
       <artifactId>jboss-logging</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
     
```

Comment:
I would like to avoid additional nose in this commit, so I created new ticket for it: NCL-908